src/main/asciidoc/_chapters/hbase-default.adoc

```diff
@@ -1949,6 +1949,34 @@ If the DFSClient configuration
 `simple`
 
 
+[[hbase.security.authentication.ui.metrics.protected]]
+*`hbase.security.authentication.ui.metrics.protected`*::
++
+.Description
+
+      Controls whether or not metrics endpoints are allowed only for admin.
+      If true, only users listed on "hbase.security.authentication.spnego.admin.users"
+      or users in group listed on "hbase.security.authentication.spnego.admin.groups"
+      are allowed to access metrics endpoints. (e.g. /jmx, /metrics, /prometheus)
+
++
+.Default
+`false`
+
+
+[[hbase.security.authentication.spnego.kerberos.endpoint.whitelist]]
+*`hbase.security.authentication.spnego.kerberos.endpoint.whitelist`*::
++
+.Description
+
+      Controls whether or not secure authentication is enabled for metrics endpoints.
+      It is valid only when "hbase.security.authentication.ui.metrics.protected" is false.
+
++
+.Default
+`/jmx,/metrics,/prometheus`
```

Comment:
IMO Defaults should be empty list to preserve default behavior. We can add a list of servlets which are supported to be whitelisted in docs